ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -89,7 +89,7 @@ public extension Colors {
             public static var backgroundDivider: UIColor = surfaceSecondary
             public static var text: UIColor = textSecondary
             public static var textDivider: UIColor = textSecondary
-            public static var textLink: UIColor = communicationBlue
+            public static var textLink = UIColor(light: Palette.communicationBlueShade10.color, dark: communicationBlue)
```

Comment:
Why isn't the `dark` side something like `dark: Palette.communicationBlue.color` ?

Reply:
`communicationBlue` is already defined as such in the Colors class, but not the `Shade10` variant